README.md

```diff
@@ -314,8 +314,9 @@ str3 = """\
 ```
 
 Any Unicode character may be used except those that must be escaped: backslash
-and the control characters (U+0000 to U+001F, U+007F). Quotation marks need not
-be escaped unless their presence would create a premature closing delimiter.
+and the control characters other than tab (U+0000 to U+0008, U+000A to U+001F,
+U+007F). Quotation marks need not be escaped unless their presence would create
```

Comment:
This still forbids line breaks, while the whole point of multi-line strings is to allow them. This is not a problem of the PR, but of the original spec. The correct wording, I believe, would be:

> ... the control characters other than tab, line feed, and carriage return (U+0000 to U+0008, U+000B,  U+000C,
> U+000E to U+001F, U+007F)....

Reply:
Good catch, @ChristianSi. I'm adding this now.

I'm still pretty green with these tools. Commit [9e46984](https://github.com/toml-lang/toml/pull/627/commits/9e469844938e64ed178732f8c4c74ca04c4ef2ed) addresses this. I used your suggestion verbatim. Can you verify it?